run-ws.py

```diff
@@ -0,0 +1,134 @@
+import os
+import json
+import socket
+import signal
+import asyncio
+import logging
+import argparse
+import websockets
+
+from jsonschema import validate
+from chainlink import Chainlink
+
+import grader.api_keys as api_keys
+
+from config import *
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger()
+
+grading_job_def = {
+    "type": "object",
+    "properties": {
+        api_keys.GRADING_JOB_ID: {"type": "string"},
+        api_keys.STAGES: {"type": "array"},
+    },
+    "required": [api_keys.GRADING_JOB_ID, api_keys.STAGES],
+    "additionalProperties": False,
+}
+
+
+async def run(token, worker_id):
```

Comment:
Nit: is there any way to break this function up a bit? It seems to be pretty long.